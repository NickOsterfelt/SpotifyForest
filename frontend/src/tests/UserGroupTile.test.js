import React from "react";
import { render } from "@testing-library/react";
import UserGroupTile from "../components/UserGroupTile";

it("renders without crashing", function () {
    render(
        <table>
            <tbody>
                <tr>
                    <UserGroupTile />
                </tr>
            </tbody>
        </table>
    );
});

it("matches snapshot", function () {
    const { asFragment } = render(
        <table>
            <tbody>
                <tr>
                    <UserGroupTile />
                </tr>
            </tbody>
        </table>
    );
    expect(asFragment()).toMatchSnapshot();
});
