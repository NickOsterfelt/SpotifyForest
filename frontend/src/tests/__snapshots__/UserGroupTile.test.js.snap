// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <table>
    <tbody>
      <tr>
        <td
          class="user-tile"
        >
          <div
            class="item"
          >
            <div
              class="header"
            >
              <h5
                href="/users/undefined"
              >
                <u />
              </h5>
            </div>
            <div
              class="image-container"
            >
              <img
                alt=""
                src="https://thumbs.dreamstime.com/b/default-avatar-profile-image-vector-social-media-user-icon-potrait-182347582.jpg"
              />
            </div>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
